tox.ini

```diff
@@ -8,6 +8,7 @@ deps =
     flake8
     pytest
     pytest-cov
+    git+https://github.com/multiscale/ymmsl-python.git@feature/checkpointing#egg=ymmsl
```

Comment:
Does this work with the latest pip? I had issues before because they removed this functionality in recent versions, but there was a work-around, IIRC some way of having GitHub make a tarball on demand which pip can then install.

Reply:
I looked into this some more, and it seems git+git has been removed, but git+https should still work. So I'm confused now as to what problems I had. If it works, it works I guess.